tests/LongRunning/SpeechRecoAuthTokenRefreshTests.ts

```diff
@@ -208,3 +208,179 @@ test("AuthToken refresh works correctly", (done: jest.DoneCallback) => {
             });
     });
 }, 1000 * 60 * 25); // 25 minutes.
+
+test("AuthToken refresh works even with fromEndpoint configurations", (done: jest.DoneCallback) => {
+    // tslint:disable-next-line:no-console
+    console.info("Name: AuthToken refresh works correctly");
+
+    if (!Settings.ExecuteLongRunningTestsBool) {
+        // tslint:disable-next-line:no-console
+        console.info("Skipping test.");
+        done();
+        return;
+    }
```

Comment:
this added test is not running in Azure DevOps?

Reply:
i hope it is part of the end-2-end testing since it takes >10minutes

Reply:
@mahilleb-msft  is there a way to specify this in the pullrequest?